submissions/Chisend/bots/temp.py

```diff
@@ -0,0 +1,198 @@
+# this is a code playground for testing things out
```

Comment:
Is this file still in use?

Reply:
No, i thought I ignored it. will update the gitignore to reflect it.

Reply:
Alright please check it out.

Reply:
Can I handle this later? So do it with the next feature and PR I open. I've added part of the giveaway feature but haven't pushed that yet.

Reply:
Alright. Can you finish up what you have to do then open the pr again. In the meantime you can move it to draft then nudge me once you set.

Reply:
Done.

Removed the temp.py